pkgs/development/libraries/libaom/default.nix

```diff
@@ -19,10 +19,8 @@ stdenv.mkDerivation rec {
     yasm perl cmake pkg-config python3
   ];
 
-  buildInputs = lib.optionals enableButteraugli [
-    libjxl
-    libhwy
```

Comment:
`libhwy` is listed as an input in the begining of file. AFAIU it's not used anymore. Worth removing?

Reply:
good catch